README.md

```diff
@@ -3,6 +3,29 @@
 rename `.env.template` to `.env.development` and fill in the missing variables. For production environement 
 create a file named `.env.production`.
 
+## Requirements to run the app
+
+To run this app you will need to have the following installed and configured on your development machine.
+
+*__Required Software:__*
+
+- [Node.js](https://nodejs.org/en/download/)
+- [yarn](https://yarnpkg.com/getting-started/install)
+- [MongoDB](https://docs.mongodb.com/manual/installation/)
+  - if you are using WSL, you will need to install MongoDB on Windows and then follow the instructions [here](https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-database#install-mongodb)
+  - altenatively, you can use a cloud MongoDB service like [MongoDB Atlas](https://www.mongodb.com/cloud/atlas)
+- [Redis](https://redis.io/download)
+
+*__Other Requirements__*
+
+- [Alchemy API key](https://www.alchemy.com/)
+  - For both Polygon and Ethereum
+- Infura API key (and secret)
+  - [Infura](https://infura.io/)
+- Etherscan API key
+  - [Etherscan](https://etherscan.io/)
+  
+
```

Comment:
Can we update the .env.template to `PORT=4500` and also change line 48 of this file to port 4500? I think that would be better otherwise api and FE run on the same port by default. I also used that port number as suggested example value on the bot to keep it consistent.

Reply:
Done.